pxd.c

```diff
@@ -182,7 +182,7 @@ static long pxd_ioctl_init(struct file *file, void __user *argp)
 	struct iov_iter iter;
 	struct iovec iov = {argp, sizeof(struct pxd_ioctl_init_args)};
 
-	iov_iter_init(&iter, WRITE, &iov, 1, sizeof(struct pxd_ioctl_init_args));
+	iov_iter_init(&iter, READ, &iov, 1, sizeof(struct pxd_ioctl_init_args));
```

Comment:
whats this change?

Reply:
@sulakshm Thank you so much!!!   Your debugging/analysis helped us look at this issue more closely and it turns out changing the initialization of the iterator to "READ" caused the runtime issue later on.  Since the  runtume assumed we started with "WRITE".     The correct fix is to reset the initializer data_source to "WRITE" so that the assumption of the original work flow is preserved https://github.com/portworx/px-fuse/pull/295/commits/81b97f6d2c225dd2ec65ebd151f0103d69415c8f.   Running the test individually now passed.

Reply:
Restarted the full BVT: https://jenkins.pwx.dev.purestorage.com/job/Dev/job/Porx-02/